mergify_engine/gitter.py

```diff
@@ -83,9 +83,18 @@ async def cleanup(self) -> None:
             self.logger.warning("git credential-cache exit fail")
         await asyncio.to_thread(shutil.rmtree, self.tmp)
 
-    async def configure(self) -> None:
-        await self("config", "user.name", f"{config.CONTEXT}-bot")
-        await self("config", "user.email", config.GIT_EMAIL)
+    async def configure(
+        self, name: typing.Optional[str] = None, email: typing.Optional[str] = None
+    ) -> None:
+        effective_name = name
+        if effective_name is None:
+            effective_name = f"{config.CONTEXT}-bot"
```

Comment:
```suggestion
        if name is None:
            name = f"{config.CONTEXT}-bot"
```
looks like it's a bit too complicated now, no? 😁 

same below

Reply:
I should stop doing JS :) I though it was enforced by mypy, but that eslint-airbnb doesn't allow changing function parameters.